lib/core/macros.nim

```diff
@@ -1692,3 +1692,39 @@ when defined(nimMacrosSizealignof):
 
 proc isExported*(n: NimNode): bool {.noSideEffect.} =
   ## Returns whether the symbol is exported or not.
+
+proc extractDocCommentsAndRunnables*(n: NimNode): NimNode {.since: (1,3,5).} =
```

Comment:
Remove this ``.since`` annotation. If it stays you cannot compile asyncmacro with ``--useVersion:1.0``.

Reply:
done